### PR TITLE
Add 'helper' blueprint

### LINKED ISSUE #191: Add HTMLbars helper blueprint
It would be useful to have possibility to add new file and add new helper, so demos could be more complete.

### PR DESCRIPTION
**Pull request by knownasilya**
This PR also adds a way to replace arbitrary properties in the blueprint
content when calling `buildProperties` in the emberCLI service.

Closes #191

Thanks @SaladFork for your Services PR, which I used to bounce off of.

### CODE REVIEW

File: app/services/ember-cli.js [hunk 1]
```diff
@@ -108,13 +112,23 @@ export default Ember.Service.extend({
...
+        });
+      }
 
       return {
         filePath: blueprint.filePath,
-        content: blueprints[blueprint.blueprint].replace(/<\%\=(.*)\%\>/gi,'')
+        content: content.replace(/<\%\=(.*)\%\>/gi,'')
       };
     }
   },
```
Comment (Gaurav0) [inline app/services/ember-cli.js]: I think this method should get a unit test.

Comment (knownasilya) [inline app/services/ember-cli.js]: Good catch

**Comment by knownasilya** [comment]
@Gaurav0 I added the unit test. Let me know if you need anything else.

File: app/gist/controller.js [hunk 2]
```diff
@@ -322,6 +322,24 @@ export default Ember.Controller.extend({
...
+      let type = 'helper'
+      let fileProperties = this.get('emberCli').buildProperties(type);
+      let filePath = prompt('File path', fileProperties.filePath);
+      let splitFilePath = filePath.split('/');
+      let file = splitFilePath[splitFilePath.length - 1];
+      let name = file.replace('.js', '').camelize();
+
+      fileProperties = this.get('emberCli').buildProperties(type, {
+        camelizedModuleName: name
+      });
```
Comment (knownasilya) [inline app/gist/controller.js]: I could see the `addFile` method being more like this in the future.

**Comment by Gaurav0** [comment]
See Travis CI Build. It thinks you are missing a semicolon.

File: app/gist/template.hbs [hunk 3]
```diff
@@ -5,13 +5,14 @@
...
+                addHelper=(action "addHelper")
                 addComponent=(action "addComponent")
                 renameFile=(action "renameFile")
                 removeFile=(action "removeFile")
+                deleteGist=(action "deleteGist")
                 saveGist="saveGist"
                 fork="fork"
                 copy="copy"
-                deleteGist=(action "deleteGist")
                 signInViaGithub="signInViaGithub"}}
```
Comment (knownasilya) [inline app/gist/template.hbs]: unfortunately the actions coming from a route cannot be passed as a callback..

**Comment by Gaurav0** [comment]
:+1:

**Pull request merged**